build.sbt

```
@@ -53,9 +53,9 @@ name := "cloudstate"
 val GrpcJavaVersion = "1.22.1"
 val GraalAkkaVersion = "0.4.1"
 val AkkaVersion = "2.5.26"
-val AkkaHttpVersion = "10.1.10+124-779795c4" // TODO: Remove once we're switching to akka-http 10.1.11
-val AkkaManagementVersion = "1.0.4"
-val AkkaPersistenceCassandraVersion = "0.96"
+val AkkaHttpVersion = "10.1.11"
+val AkkaManagementVersion = "1.0.5"
+val AkkaPersistenceCassandraVersion = "0.100"
```

Use [`0.102`](https://github.com/akka/akka-persistence-cassandra/releases/tag/v0.102) already?

```suggestion
val AkkaPersistenceCassandraVersion = "0.102"
```